packages/manager/apps/hub/src/components/hub-order-tracking/HubOrderTracking.component.tsx

```diff
@@ -192,15 +201,15 @@ export default function HubOrderTracking() {
                       color={ODS_THEME_COLOR_INTENT.primary}
                       className="inline-block mr-1"
                     >
-                      <strong>{format(new Date(currentStatus.date))}</strong>
-                      &nbsp;{t(`order_tracking_history_${currentStatus.label}`)}
+                      <strong>{format(new Date(displayedStatus.date))}</strong>
+                      &nbsp;{t(`order_tracking_history_${displayedStatus.label}`)}
                     </OsdsText>
                     <span className="inline-block size-[16px]">
                       <OsdsIcon
                         size={ODS_ICON_SIZE.xxs}
                         color={ODS_THEME_COLOR_INTENT.text}
                         name={
-                          !ERROR_STATUS.includes(currentStatus.label) &&
+                          !ERROR_STATUS.includes(displayedStatus.label) &&
```

Comment:
`displayedStatus` can be undefined
```suggestion
                          !ERROR_STATUS.includes(displayedStatus?.label) &&
```